app/controllers/stores_controller.rb

```diff
@@ -1,22 +1,20 @@
 class StoresController < ApplicationController
 
-  def index
-
-
-  end
-
   def show
     @store = Store.find_or_create_by(params[:display_address])
   end
 
   def search
     parameters = { term: params[:term], category_filter: params[:category_filter], limit: 5, location: params[:location] }
     results = Yelp.client.search(params[:location], parameters)
+    @stores = []
     results.businesses.each do |business|
-      store = Store.find_by(address: business.location.display_address)
+      store = Store.find_by(address: business.id)
         unless store
-          Store.create(name: business.name, address: business.location.display_address[0])
+          store = Store.new(name: business.name, address: business.id)
```

Comment:
Please include the rest of the store factors that we initially included yesterday -display_phone, email, and categories as description. Thanks